magento/pwa-studio
# PWA-213: [UNIT TEST] Email Validation on Signin, Create Acount, Forgot Password

## Description

Reviewed this task with team lead, and it turns out this feature should have never been implemented in the first place. There is little value in doing client-side validation, and the most we should do in our app is validate the email address isn't in use, which isn't currently possible with the form library we're using. He requested we strip this out, which we're doing in the scope of this task now. Article for reference: https://medium.com/hackernoon/the-100-correct-way-to-validate-email-addresses-7c4818f24643

## Related Issue





* [[PWA-213](https://jira.corp.magento.com/browse/PWA-213)] [UNIT TEST] Email Validation on Signin, Create Acount, Forgot Password

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


For each of the forms that had this validation (SignIn, Forgot Password, Create Account, Checkout Shipping/Billing/Payment)
1. Submit clean form and verify required validation still fires
2. Enter an "invalid" email address in email address field and verify it no longer complains (eg. `test@domain`)

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
